ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -43,3 +44,8 @@ tests:
   commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-frontend.sh
   container:
     from: test-bin
+- artifact_dir: /tmp/artifacts
+  as: e2e
```

Comment:
I'd give this a more specific name like console-e2e. This might get confused with aws-e2e otherwise